src/main/java/com/greenfox/poker/model/Game.java

```diff
@@ -0,0 +1,121 @@
+package com.greenfox.poker.model;
+
+
+import java.util.List;
+
+public class Game {
```

Comment:
There are much more fields defined here that we need according to our current specification. You should update this model.